man/systemd.network.xml

```diff
@@ -1260,6 +1260,14 @@
             should have lower costs.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Priority=</varname></term>
+          <listitem>
+            <para>Sets the "priority" of sending packets on this interface.
+            Each port in a bridge may have a different priority which is used
+            to decide which link to use.</para>
```

Comment:
please document here that this an integer, which range is permissible and whether the higher priority is referred to by higher or lower numbers.